web/packages/test/scripts/configure-bridgehub.sh

```diff
@@ -69,7 +69,17 @@ open_hrmp_channels()
     open_hrmp_channel "${relaychain_ws_url}" "${relaychain_sudo_seed}" 2000 1000 8 512 # Assethub -> Penpal
 }
 
+set_gateway()
+{
+    echo "Setting gateway contract"
+    local storage_key=$(echo $GATEWAY_STORAGE_KEY | cut -c3-)
+    local gateway=$(echo $GATEWAY_PROXY_CONTRACT | cut -c3-)
+    local transact_call="0x00040440"$storage_key"50"$gateway
```

Comment:
Cool!  I would assume it's a `System::set_storage()` call here, Just curious how to generate the `storage_key` here.

Reply:
So basically I modified this unit test to print it out. The generation of the key looks stable.

https://github.com/Snowfork/polkadot-sdk/pull/66/files#diff-79a28d4d3e1749050341c2424f00c4c139825b1a20937767f83e58b95166735cR153

The generation of this key is stable across all runtimes:
https://github.com/Snowfork/polkadot-sdk/blob/alistair/gateway-in-storage/substrate/frame/support/src/lib.rs#L333